src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptEditorImprovementsTest.java

```diff
@@ -138,6 +138,8 @@ private void testNestedQuotesDisabled(char outerQuote, char innerQuote) {
 
         // Disable string literal improvements
         getClientConfigurationSettings(languageServer).editor.enableStringLiteralImprovements = false;
+        // Bump the modification count since we changed settings directly
```

Comment:
Now that config is cached aggressively, we must explicitly bump the language server definition-level modification count after making explicit client config changes. This will be a recurring change below.